CLAUDE.md

```diff
@@ -0,0 +1,56 @@
+# Claude Development Guidelines
+
+This file contains specific guidelines for working with Claude on the Alchemy repository.
+
+## Development Commands
```

Comment:
Ugh, why are you writing this file? Please delete it and then merge from main. I've already implemented a good one